zigpy/zcl/__init__.py

```diff
@@ -35,6 +29,12 @@ def __init__(cls, name, bases, nmspc):  # noqa: N805
                 command_name, schema, is_reply = details
                 cls._client_command_idx[command_name] = command_id
 
+        if not getattr(cls, '_skip_registry', False):
```

Comment:
Can you keep the structure as it was before, but just move it down here?

Reply:
Done!